test/reaping-faulty-nodes.js

```diff
@@ -217,10 +217,21 @@ test2('tombstone should be gossiped with flag when applied as state', getCluster
             // clear all pings that happened before we gossiped the tombstone
             dsl.consumePings(t, tc),
 
-            // Wait for a ping from the SUT and validate that it does not gossip about the tombstone
-            dsl.validateEventBody(t, tc, {
-                type: events.Types.Ping,
-                direction: 'request'
+            // Wait for a ping from the SUT and validate that it does not gossip about the tombstone.
```

Comment:
Good remark. I refactored the checking part to a separate function and updated the comment.